x-pack/plugins/license_management/public/components/telemetry_opt_in/telemetry_opt_in.js

```diff
@@ -60,7 +61,12 @@ export class TelemetryOptIn extends React.Component {
       toCurrentCustomers = (
         <Fragment>
           <EuiText>
-            <p>Gold and platinum customers: help support give you better service.</p>
+            <p>
+              <FormattedMessage
+                id="xpack.licenseMgmt.telemetryOptIn.customersHelpSupportDescription"
+                defaultMessage="Gold and platinum customers: help support give you better service."
```

Comment:
I'm not sure if license types (gold, platinum) should be translated here.  Are we translating them other places where we refer to license types?

Reply:
Chi-Wei Chang in slack localization chat was answered that we need to translate license types